### ISSUE TITLE
How is this case handled in evaluation?

### ISSUE THREAD

**Issue by martmichals** [sheryc/QEN#2]
I created a visualization of the train/validation/test sets used in your work. This is a small snippet of that visualization, with train/validation/test corresponding to black/green/purple. If the validation set is withheld and nodes are independently placed into the remaining taxonomy, there is no possibility of "iced coffee" being placed correctly into this taxonomy, artificially lowering performance metrics as described in the paper (since they rely on the parent to determine placement).

Is this a true issue, or is this handled somewhere I have not noticed in the codebase?

<img width="513" alt="image" src="https://user-images.githubusercontent.com/49954516/171449082-5462c858-ae17-4d0b-94eb-8ee7b1886d7c.png">
